Item Stack, Set Max Value

Songbird098 opened the pull request:
In Objects Helper, set ItemSpinBox's max value to the stack size of selected item. Result: You cannot set the number of items per stack higher than pre-defined stack size for the item (This happens in signal: _item_dropdown_selected)

Songbird098 committed:
Item Stack, Set Max Value

In Objects Helper, set ItemSpinBox's max value to the stack size of selected item.
Result: You cannot set the number of items per stack higher than pre-defined stack size for the item
(This happens in signal: _item_dropdown_selected)

this-is-envy reviewed (changes requested):
A bunch of random files got added from (I think?) VisualStudio that need to be removed

- Scripts/.vs/Scripts/FileContentIndex/<uuid>.vsidx
- Scripts/.vs/Scripts/v17/.wsuo
- Scripts/.vs/VSWorkpsaceState.json
- Scripts/.vs/slnx.sqlite

Otherwise only minor comments. I did see some changes I don't understand in ObjectHelper.tscn but I don't actually think it's related to your change so I'll poke around with that and try to figure it out once the visual studio stuff is gone and this is back up for review.

CI (Continuous Integration) jobs are failing which will need to be fixed. You can either view the output on this PR here:

![image](https://github.com/user-attachments/assets/8d44764b-cc81-4f5a-97b8-90eb5c9f59b1)


Or run it locally (this is my preference as it's faster/easier iteration) and that process is documented here: https://github.com/Small-Loan-Studio/TGO/blob/tgo_dev/docs/docker.md

Songbird098 committed:
PR Feedback

Songbird098 committed:
PR Lint & Format 1

Songbird098 committed:
PR Lint & Format 2

Songbird098 committed:
PR Lint & Format 3

Post Docker Solution

Found:
Always press space twice after code for commenting
(code  #like this)
